HoloPipelines/core/wrappers/obj2gltf.py

```diff
@@ -2,14 +2,14 @@
 import subprocess
 
 
-def call_obj2gltf(obj_input_path: str, glb_output_path: str):
+def call_obj2gltf(obj_input_filepath: str, glb_output_file_path: str) -> None:
```

Comment:
filepath or file_path?

Reply:
stick to `file_path`, it seems to be used more commonly in the rest of the codebase